f8a_jobs/api_spec.yaml

```
@@ -177,6 +245,9 @@ components:
         flow_name:
           type: string
           example: <flow_name>
+        source:
+          type: string
+          example: <name_of_consumer>
```

If you set of sources, then use enum
```suggestion
          type: string
          enum:
             - api
             - worker
          example: <name_of_consumer>
```

---

Also you can specify default value